ONSdigital/census-rm-toolbox
# Process bulk request refusals

# Motivation and Context
We need a way of turning files of bulk refusals containing case ID's and refusal types into RM messages and feeding back any validation failures. The aim is also to make this a reusable 'bulk processing' framework so that ways of processing other file/message types are easy to add in the future.

# What has changed
* Add bulk processing module
* Add refusal processor and tests
* Close connections in the DB helper methods
* Update README

# How to test?
Build and push a tagged version of toolbox on this branch and terraform/apply using the branches attached to the card. Then run the bulk refusals processor and check it can successfully refuse any good rows and feeds back appropriate errors for any incorrect rows (bad refusal types, bad case ID's, bad file format etc.)

# Links
https://trello.com/c/I3pciSPB/1116-process-bulk-request-refusals-13
